### ISSUE TITLE
dyanamic window state should support windows split with aero snap

### ISSUE THREAD

**Issue by dmwyatt** [TranslucentTB/TranslucentTB#116]
When you have two windows split it's an equivalent situation to having one maximized window and dynamic-ws should support that.  Right?

**Comment by sylveon** [comment]
Yes, but there is no way to detect Aero Snap, and manually doing this kind of check would be expensive, buggy and require quite a lot of work.

**Comment by denosawr** [comment]
@FSund has suggested this link: https://stackoverflow.com/a/47911169/1850917

**Comment by denosawr** [comment]
My opinion on it would be that it would require quite a bit of processing power to find the location of each window. However, it seems like a valid way to approach the issue, and the issue with processing power could be mostly mitigated when/if polling is implemented.

**Comment by denosawr** [comment]
Actually, on a reread, it seems that we could wait for a WM_WINDOWPOSCHANGED signal. However, i still think this should be a V5 thing. (I’m on mobile and can’t seem to set milestones, but it would be appreciated if someone could help me out)

**Comment by redbrain1014** [comment]
Interested in this feature. Would you consider increasing the priority or adding it to the version 5 project?

**Comment by sylveon** [comment]
Still no way to reliably detect aero snap.

**Comment by redbrain1014** [comment]
😕 :shrugs: Oh well.

**Comment by sylveon** [comment]
Update 4 years later: now you can. https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-iswindowarranged

**Comment by EmErDogi** [comment]
Any updates?
